index.js

```diff
@@ -240,6 +241,11 @@ class Ky {
 		if (this._options.searchParams) {
 			const url = new URL(this.request.url);
 			url.search = new URLSearchParams(this._options.searchParams);
+
+			if (((supportsFormData && this._options.body instanceof globals.FormData) || this._options.body instanceof URLSearchParams) && !(this._options.headers && this._options.headers['content-type'])) {
```

Comment:
Leave a code comment that links to the issue explaining this hack, as it's not obvious to new contributors why we do this.